caveman/build.gradle

```diff
@@ -0,0 +1,128 @@
+apply plugin: 'com.android.library'
+apply plugin: 'com.github.dcendents.android-maven'
+apply plugin: 'com.jfrog.bintray'
+
+ext {
+    bintrayRepo = 'maven'
+    bintrayName = 'caveman'
+
+    publishedGroupId = 'au.com.outware'
+    libraryName = 'caveman'
+    artifact = 'caveman'
+
+    libraryDescription = 'Caveman is a real-time environment manager for Android applications.\n' +
+            '\n' +
+            'This serves a similar purpose to setting build variables for different variants (with ' +
+            'support for booleans, numbers and strings), however the benefit of this approach is ' +
+            'that you can use one application for all environments instead of one application per ' +
+            'build variant. Another benefit is that values can be updated without requiring a new ' +
+            'build, which is particularly useful as this means values can be updated by testers, ' +
+            'product managers etc.'
+
+    siteUrl = 'https://github.com/outware/caveman/'
+    gitUrl = 'https://github.com/outware/caveman.git'
+
+    libraryVersion = '1.0.0'
+
+    developerId = 'outware-mobile'
+    developerName = 'Outware Mobile'
+    developerEmail = 'technology@outware.com.au'
+
+    licenseName = 'MIT'
+    licenseUrl = 'https://opensource.org/licenses/MIT'
+    allLicenses = ["MIT"]
+}
+
+android {
+    compileSdkVersion 23
+    buildToolsVersion "23.0.1"
+
+    defaultConfig {
+        minSdkVersion 14
```

Comment:
16 is normally my go-to, but Branko wanted to go with 14 to maximise compatibility. With the usage of the support libraries, theres nothing stopping us from going back this far and we could probably even go back further if we wanted to